code/game/objects/items/devices/radio/radio.dm

```diff
@@ -186,6 +186,20 @@
 				. = TRUE
 
 /obj/item/radio/talk_into(atom/movable/M, message, channel, list/spans, datum/language/language, list/message_mods)
+	if(HAS_TRAIT(M, TRAIT_SIGN_LANG)) //Forces Sign Language users to wear the translation gloves to speak over radios
+		var/mob/living/carbon/mute = M
+		if(istype(mute))
```

Comment:
As far as I've been able to test, cyborgs are under living/silicon which makes them immune to the effect, which is good because this effect is tied to a tongue, which cyborgs do not possess, instead using their speakers connected to the MMI. Meanwhile, MMIs & Posibrains count as items, which makes them unable to get said trait, and further be unaffected by said trait.
If you meant something else by that, it went over my head, and I apologize

Reply:
Nah you're fine